q2_demux/_summarize/_visualizer.py

```diff
@@ -108,6 +123,7 @@ def summarize(output_dir: str, data: _PlotQualView, n: int=10000) -> None:
         count = 0
         for seq in _read_fastq_seqs(file):
             count += 1
+
```

Comment:
This isn't the boundary of a function though, the implementation continues below. It's not a big deal having it either way.